README.md

```diff
@@ -83,20 +88,29 @@ var app = express();
 ```
 
 ```js
-// Your answer...
+app.get(req, res){
+  respond.send("index")});
+app.put(req, res){
+  respond.send("create")});
+app.post(req, res){
+  respond.send("update")});
+app.delete(req, res){
+  respond.send("delete")});
```

Comment:
These responses need to use the argument for the function that corresponds to the response.  There is no variable named `response`.